intermediate_notebooks/E2E/taxi/NYCTaxi-E2E.ipynb

```diff
@@ -473,23 +944,25 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 27,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "2.112081468833012"
```

Comment:
same comment as for timing the ETL. When you run against the full dataset, the RMSE is significantly lower.